include/fmt/color.h

```diff
@@ -209,6 +215,126 @@ struct rgb {
   uint8_t b;
 };
 
+// Experimental text formatting support.
+class text_format {
```

Comment:
`format` is a bit of an overloaded term in {fmt}. Can we choose some other name for this struct like `style`?